src/main/com/fulcrologic/fulcro/rendering/ident_optimized_render.cljc

```diff
@@ -79,13 +77,10 @@
     (let [{:com.fulcrologic.fulcro.application/keys [runtime-atom]} app
           {:com.fulcrologic.fulcro.application/keys [indexes]} @runtime-atom
           {:keys [prop->classes class->components]} indexes]
-      (reduce
-        (fn [result prop]
-          (let [classes    (prop->classes prop)
-                components (reduce #(into %1 (class->components %2)) #{} classes)]
-            (into result components)))
-        #{}
-        property-set))))
+      (into #{}
+            (comp (mapcat prop->classes)
+                  (mapcat class->components))
+            property-set))))
```

Comment:
This seems an improvement. Realize that property-set is usually one thing...maybe a few. Rarely more than 3.